tools/ci-scripts/linux/image-builder/build_boosts.zsh

```diff
@@ -2,7 +2,7 @@
 
 set -eux
 
-BOOST_VERSIONS=(1.58.0 1.59.0 1.60.0 1.61.0 1.62.0 1.63.0 1.64.0 1.65.1)
+BOOST_VERSIONS=(1.58.0 1.59.0 1.60.0 1.61.0 1.62.0 1.63.0 1.64.0 1.65.1 1.66.0)
```

Comment:
Should we consider deprecating one of the older versions whenever we introduce the latest one?

Reply:
That's https://github.com/Microsoft/bond/issues/771